mdoc/src/main/scala/mdoc/internal/cli/MainOps.scala

```diff
@@ -121,7 +121,14 @@ final class MainOps(
   }
 
   def generateCompleteSite(): Exit = {
-    val files = IO.inputFiles(settings)
+    val inIsFile = settings.in.toFile.isFile
```

Comment:
nit: for some reason I try to use `isX` as the naming convention for boolean (it's not always possible)

```suggestion
    val isFile = settings.in.isFile
```

Reply:
Sure, that makes sense. Changed.